test/controllers/questions_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class QuestionsControllerTest < ActionController::TestCase
+  setup do
+    @question = questions(:one)
+  end
+
+  test "should get index" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.